tripal/src/Services/TripalPublish.php

```diff
@@ -477,9 +477,10 @@ protected function getEntityTitles($matches) {
           $field = $this->field_info[$field_name]['instance'];
           $main_prop = $field->mainPropertyName();
           $value = $match[$field_name][$delta][$main_prop]['value']->getValue();
-          if ($value !== NULL) {
-            $entity_title = trim(preg_replace("/\[$field_name\]/", $value,  $entity_title));
+          if ($value === NULL) {
+            $value = '';
           }
```

Comment:
```suggestion
          if ($value === NULL) {
            $value = '';
          }
```
```suggestion
          if ($value === NULL) {
            $value = '';
          }
```
```suggestion
          $value = '';
          if (array_key_exists($delta, $match[$field_name])) {
            $value = $match[$field_name][$delta][$main_prop]['value']->getValue();
          }
          if ($value === NULL) {
            $value = '';
          }
```
With this change I can publish an organism.